### PR TITLE
Update description

### PR DESCRIPTION
**Pull request by Take-A-Byte**
Adding description to change it on nuget.org

### CODE REVIEW

**Review by erhard-pspdfkit — APPROVED** [review]

File: nuget/pspdfkit-ios-instant.nuspec [hunk 1]
```diff
@@ -4,16 +4,18 @@
...
         <license type="file">LICENSE.md</license>
-        <projectUrl>https://pspdfkit.com/instant/</projectUrl>
+        <projectUrl>https://nutrient.io/instant/</projectUrl>
         <repository type="git" url="https://github.com/PSPDFKit/Xamarin-iOS"/>
         <icon>images\icon.png</icon>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>
             The leading framework for displaying, annotating and editing PDFs on iOS, Android, Windows, Electron and the Web.
+
+            The PSPDFKit Xamarin SDK is being sunset and replaced Nutrient .NET for iOS (formally PSPDFKit .NET for iOS). All current Xamarin licenses are valid and supported through Dec 1, 2025, or until they expire—whichever comes later. The migration from Xamarin to .NET is straightforward, as explained in the official Microsoft guide. Interested in transitioning to another Nutrient product? Contact us and we are happy to help.
```
Comment (deutschbitte) [inline nuget/pspdfkit-ios-instant.nuspec]:
```suggestion
            Nutrient Xamarin SDK (formerly PSPDFKit Xamarin for iOS and PSPDFKit Xamarin for Android) is being sunset and replaced by [Nutrient .NET for iOS](https://www.nutrient.io/guides/ios/dotnet/) and [Nutrient .NET for Android](https://www.nutrient.io/guides/android/dotnet/). All Nutrient Xamarin licenses will remain valid and fully supported until 1 December 2025, or until your license expires, whichever occurs later. Interested in transitioning to another Nutrient product? [Contact us](https://www.nutrient.io/company/contact) and we’ll be happy to help.
```

File: nuget/pspdfkit-ios-model.nuspec [hunk 2]
```diff
@@ -4,16 +4,18 @@
...
         <license type="file">LICENSE.md</license>
-        <projectUrl>https://pspdfkit.com/</projectUrl>
+        <projectUrl>https://nutrient.io/</projectUrl>
         <repository type="git" url="https://github.com/PSPDFKit/Xamarin-iOS"/>
         <icon>images\icon.png</icon>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>
             The leading framework for displaying, annotating and editing PDFs on iOS, Android, Windows, Electron and the Web.
+            
+            The PSPDFKit Xamarin SDK is being sunset and replaced Nutrient .NET for iOS (formally PSPDFKit .NET for iOS). All current Xamarin licenses are valid and supported through Dec 1, 2025, or until they expire—whichever comes later. The migration from Xamarin to .NET is straightforward, as explained in the official Microsoft guide. Interested in transitioning to another Nutrient product? Contact us and we are happy to help.
```
Comment (deutschbitte) [inline nuget/pspdfkit-ios-model.nuspec]:
```suggestion
            Nutrient Xamarin SDK (formerly PSPDFKit Xamarin for iOS and PSPDFKit Xamarin for Android) is being sunset and replaced by [Nutrient .NET for iOS](https://www.nutrient.io/guides/ios/dotnet/) and [Nutrient .NET for Android](https://www.nutrient.io/guides/android/dotnet/). All Nutrient Xamarin licenses will remain valid and fully supported until 1 December 2025, or until your license expires, whichever occurs later. Interested in transitioning to another Nutrient product? [Contact us](https://www.nutrient.io/company/contact) and we’ll be happy to help.
```

File: nuget/pspdfkit-ios-ui.nuspec [hunk 3]
```diff
@@ -4,16 +4,18 @@
...
         <license type="file">LICENSE.md</license>
-        <projectUrl>https://pspdfkit.com/</projectUrl>
+        <projectUrl>https://nutrient.io/</projectUrl>
         <repository type="git" url="https://github.com/PSPDFKit/Xamarin-iOS"/>
         <icon>images\icon.png</icon>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>
             The leading framework for displaying, annotating and editing PDFs on iOS, Android, Windows, Electron and the Web.
+
+            The PSPDFKit Xamarin SDK is being sunset and replaced Nutrient .NET for iOS (formally PSPDFKit .NET for iOS). All current Xamarin licenses are valid and supported through Dec 1, 2025, or until they expire—whichever comes later. The migration from Xamarin to .NET is straightforward, as explained in the official Microsoft guide. Interested in transitioning to another Nutrient product? Contact us and we are happy to help.
```
Comment (deutschbitte) [inline nuget/pspdfkit-ios-ui.nuspec]:
```suggestion
            Nutrient Xamarin SDK (formerly PSPDFKit Xamarin for iOS and PSPDFKit Xamarin for Android) is being sunset and replaced by [Nutrient .NET for iOS](https://www.nutrient.io/guides/ios/dotnet/) and [Nutrient .NET for Android](https://www.nutrient.io/guides/android/dotnet/). All Nutrient Xamarin licenses will remain valid and fully supported until 1 December 2025, or until your license expires, whichever occurs later. Interested in transitioning to another Nutrient product? [Contact us](https://www.nutrient.io/company/contact) and we’ll be happy to help.
```

File: nuget/pspdfkit-mac-model.nuspec [hunk 4]
```diff
@@ -4,16 +4,18 @@
...
         <license type="file">LICENSE.md</license>
-        <projectUrl>https://pspdfkit.com/</projectUrl>
+        <projectUrl>https://nutrient.io/</projectUrl>
         <repository type="git" url="https://github.com/PSPDFKit/Xamarin-iOS"/>
         <icon>images\icon.png</icon>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>
             The leading framework for displaying, annotating and editing PDFs on iOS, Android, Windows, Electron and the Web.
+
+            The PSPDFKit Xamarin SDK is being sunset and replaced Nutrient .NET for iOS (formally PSPDFKit .NET for iOS). All current Xamarin licenses are valid and supported through Dec 1, 2025, or until they expire—whichever comes later. The migration from Xamarin to .NET is straightforward, as explained in the official Microsoft guide. Interested in transitioning to another Nutrient product? Contact us and we are happy to help.
```
Comment (deutschbitte) [inline nuget/pspdfkit-mac-model.nuspec]:
```suggestion
            Nutrient Xamarin SDK (formerly PSPDFKit Xamarin for iOS and PSPDFKit Xamarin for Android) is being sunset and replaced by [Nutrient .NET for iOS](https://www.nutrient.io/guides/ios/dotnet/) and [Nutrient .NET for Android](https://www.nutrient.io/guides/android/dotnet/). All Nutrient Xamarin licenses will remain valid and fully supported until 1 December 2025, or until your license expires, whichever occurs later. Interested in transitioning to another Nutrient product? [Contact us](https://www.nutrient.io/company/contact) and we’ll be happy to help.
```

File: nuget/pspdfkit-ios-instant.nuspec (hunk 1 above)
Comment (Take-A-Byte) [reply nuget/pspdfkit-ios-instant.nuspec]: @deutschbitte Please check out the comment [here](https://github.com/PSPDFKit/Xamarin-Android/pull/26#discussion_r1821941784). 
Would this work?
```suggestion
            The PSPDFKit Xamarin SDK is being sunset and replaced by Nutrient .NET for iOS (formally PSPDFKit .NET for iOS). All Xamarin SDK licenses will remain valid and fully supported until 1 December 2025, or until your license expires, whichever occurs later. Interested in transitioning to another Nutrient product? Contact us and we’ll be happy to help.
```

File: nuget/pspdfkit-ios-model.nuspec (hunk 2 above)
Comment (Take-A-Byte) [reply nuget/pspdfkit-ios-model.nuspec]: Would this work?
```suggestion
            The PSPDFKit Xamarin SDK is being sunset and replaced by Nutrient .NET for iOS (formally PSPDFKit .NET for iOS). All Xamarin SDK licenses will remain valid and fully supported until 1 December 2025, or until your license expires, whichever occurs later. Interested in transitioning to another Nutrient product? Contact us and we’ll be happy to help.
```

File: nuget/pspdfkit-ios-ui.nuspec (hunk 3 above)
Comment (Take-A-Byte) [reply nuget/pspdfkit-ios-ui.nuspec]: Would this work?
```suggestion
            The PSPDFKit Xamarin SDK is being sunset and replaced by Nutrient .NET for iOS (formally PSPDFKit .NET for iOS). All Xamarin SDK licenses will remain valid and fully supported until 1 December 2025, or until your license expires, whichever occurs later. Interested in transitioning to another Nutrient product? Contact us and we’ll be happy to help.
```

File: nuget/pspdfkit-mac-model.nuspec (hunk 4 above)
Comment (Take-A-Byte) [reply nuget/pspdfkit-mac-model.nuspec]: Would this work?
```suggestion
            The PSPDFKit Xamarin SDK is being sunset and replaced by Nutrient .NET for iOS (formally PSPDFKit .NET for iOS). All Xamarin SDK licenses will remain valid and fully supported until 1 December 2025, or until your license expires, whichever occurs later. Interested in transitioning to another Nutrient product? Contact us and we’ll be happy to help.
```

**Pull request merged**